client/src/modules/debtors/groups.update.html

```diff
@@ -23,9 +23,19 @@
 
           <div
             class="form-group has-feedback"
-            ng-class="{'has-error' : debtorGroup.color.$invalid && debtorGroup.$submitted}">
+            ng-class="{'has-error' : !GroupUpdateCtrl.group.color && debtorGroup.$submitted}">
             <label class="control-label" translate>FORM.LABELS.COLOR</label>
-            <input name="color" id="title-color" type="color" ng-model="GroupUpdateCtrl.group.color" class="form-control" ng-value="GroupUpdateCtrl.group.color">
+
+            <ui-select tagging ng-model="GroupUpdateCtrl.group.color">
```

Comment:
I don't think you need the `tagging` directive here.  I'm not even sure what it does.